### PR TITLE
CSM deploy csv updates - addition of MET sensors

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Addition of MET sensors to deploy csvs per Redmine ticket #15493.  
Refer to Google sheet for sensors added: https://docs.google.com/spreadsheets/d/1u74ZBrWE5pZw9_auFhBxFku91GOYXml9geBjmnQ8cLw/edit#gid=1801947899 (METBK-CT PIONEER Deploy csvs tab)
Also - to verify MET sensor heights refer to attached file - highlighted column D: METBK_Module_Heights_RGT_2023-06-27.xlsx

[METBK_Module_Heights_RGT_2023-06-27.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/12409470/METBK_Module_Heights_RGT_2023-06-27.xlsx)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Look into ingesting METBK-CT recovered instrument data (Redmine #14304, #15493) [DT-180]](https://trello.com/c/sSSfBQmE/180-look-into-ingesting-metbk-ct-recovered-instrument-data-redmine-14304-15493-dt-180)

File: deployment/CP01CNSM_Deploy.csv [hunk 1]
```diff
@@ -537,9 +747,23 @@ AR66,,AR70,,CP01CNSM-SBD11-00-DCLENG000,16,1,2022-04-10T14:35:00,2022-11-11T13:2
...
+AR66,,AR70,,CP01CNSM-SBD12-06-METBKA002,16,1,2022-04-10T14:35:00,2022-11-11T13:21:00,CGMCP-01CNSM-00016,,CGINS-METBPR-00250,40.13325,-70.778317,,-4,135,,,
+AR66,,AR70,,CP01CNSM-SBD12-06-METBKA003,16,1,2022-04-10T14:35:00,2022-11-11T13:21:00,CGMCP-01CNSM-00016,,CGINS-METHRH-00295,40.13325,-70.778317,,-4,135,,,
+AR66,,AR70,,CP01CNSM-SBD12-06-METBKA004,16,1,2022-04-10T14:35:00,2022-11-11T13:21:00,CGMCP-01CNSM-00016,,CGINS-METLWR-00285,40.13325,-70.778317,,-4,135,,,
+AR66,,AR70,,CP01CNSM-SBD12-06-METBKA005,16,1,2022-04-10T14:35:00,2022-11-11T13:21:00,CGMCP-01CNSM-00016,,CGINS-METPRC-00254,40.13325,-70.778317,,-4,135,,,
+AR66,,AR70,,CP01CNSM-SBD12-06-METBKA006,16,1,2022-04-10T14:35:00,2022-11-11T13:21:00,CGMCP-01CNSM-00016,,CGINS-METSWR-00294,40.13325,-70.778317,,-4,135,,,
+AR66,,AR70,,CP01CNSM-SBD12-06-METBKA007,16,1,2022-04-10T14:35:00,2022-11-11T13:21:00,CGMCP-01CNSM-00016,,CGINS-METWND-00251,40.13325,-70.778317,,-5,135,,,
 AR66,,AR70,,CP01CNSM-SBD12-08-FDCHPA000,16,1,2022-04-10T14:35:00,2022-11-11T13:21:00,CGMCP-01CNSM-00016,,CGINS-FDCHPB-11802,40.13325,-70.778317,,-5,135,,,
```
Comment (cooleyky) [inline deployment/CP01CNSM_Deploy.csv:769]: Looks good.

**Review by cooleyky — APPROVED** [review]
All changes look good

File: deployment/CP03ISSM_Deploy.csv [hunk 2]
```diff
@@ -151,6 +179,13 @@ AR8,,AR18,,CP03ISSM-SBD11-00-DCLENG000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:
...
+AR8,,AR18,,CP03ISSM-SBD11-06-METBKA001,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-METCTD-11117,40.36192,-70.87827,,1,92,,,
+AR8,,AR18,,CP03ISSM-SBD11-06-METBKA002,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-METBPR-00254,40.36192,-70.87827,,-4,92,,,
+AR8,,AR18,,CP03ISSM-SBD11-06-METBKA003,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-METHRH-00277,40.36192,-70.87827,,-4,92,,,
+AR8,,AR18,,CP03ISSM-SBD11-06-METBKA004,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-METLWR-00273,40.36192,-70.87827,,-4,92,,,
+AR8,,AR18,,CP03ISSM-SBD11-06-METBKA005,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-METPRC-00254,40.36192,-70.87827,,-4,92,,,
+AR8,,AR18,,CP03ISSM-SBD11-06-METBKA006,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-METSWR-00264,40.36192,-70.87827,,-4,92,,,
+AR8,,AR18,,CP03ISSM-SBD11-06-METBKA007,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-METWND-00264,40.36192,-70.87827,,-5,92,,,
```
Comment (sbeaulieu) [inline deployment/CP03ISSM_Deploy.csv:188]: I'd like to check if coincidence same sensor UID 00264

File: deployment/CP03ISSM_Deploy.csv [hunk 3]
```diff
@@ -431,6 +522,13 @@ AR61,,AR66,,CP03ISSM-SBD11-00-DCLENG000,14,1,2021-11-01T14:10:00,2022-04-13T21:2
...
 AR61,,AR66,,CP03ISSM-SBD11-02-HYDGN0000,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGINS-HYDGN0-41692,40.3646,-70.888252,,0,93,,,
 AR61,,AR66,,CP03ISSM-SBD11-06-METBKA000,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGINS-METLGR-00033,40.3646,-70.888252,,0,93,,,
+AR61,,AR66,,CP03ISSM-SBD11-06-METBKA001,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGINS-METCTD-11001,40.3646,-70.888252,,1,93,,,
+AR61,,AR66,,CP03ISSM-SBD11-06-METBKA002,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGINS-METBPR-00272,40.3646,-70.888252,,-4,93,,,
+AR61,,AR66,,CP03ISSM-SBD11-06-METBKA003,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGINS-METHRH-00362,40.3646,-70.888252,,-4,93,,,
+AR61,,AR66,,CP03ISSM-SBD11-06-METBKA004,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGINS-METLWR-00273,40.3646,-70.888252,,-4,93,,,
+AR61,,AR66,,CP03ISSM-SBD11-06-METBKA005,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGINS-METPRC-00273,40.3646,-70.888252,,-4,93,,,
```
Comment (sbeaulieu) [inline deployment/CP03ISSM_Deploy.csv:529]: similar to above I'd like to check 00273

File: deployment/CP04OSSM_Deploy.csv [hunk 4]
```diff
@@ -463,6 +561,13 @@ AR66,,AR70,,CP04OSSM-SBD11-00-DCLENG000,15,1,2022-04-09T14:56:00,2022-11-10T13:3
...
+AR66,,AR70,,CP04OSSM-SBD11-06-METBKA004,15,1,2022-04-09T14:56:00,2022-11-10T13:36:00,CGMCP-04OSSM-00015,,CGINS-METLWR-00295,39.937517,-70.88685,,-4,454,,,
+AR66,,AR70,,CP04OSSM-SBD11-06-METBKA005,15,1,2022-04-09T14:56:00,2022-11-10T13:36:00,CGMCP-04OSSM-00015,,CGINS-METPRC-00263,39.937517,-70.88685,,-4,454,,,
+AR66,,AR70,,CP04OSSM-SBD11-06-METBKA006,15,1,2022-04-09T14:56:00,2022-11-10T13:36:00,CGMCP-04OSSM-00015,,CGINS-METSWR-00262,39.937517,-70.88685,,-4,454,,,
+AR66,,AR70,,CP04OSSM-SBD11-06-METBKA007,15,1,2022-04-09T14:56:00,2022-11-10T13:36:00,CGMCP-04OSSM-00015,,CGINS-METWND-00258,39.937517,-70.88685,,-5,454,,,
 AR66,,AR70,,CP04OSSM-SBD12-00-DCLENG000,15,1,2022-04-09T14:56:00,2022-11-10T13:36:00,CGMCP-04OSSM-00015,,3718-00001-00001-00051,39.937517,-70.88685,,0,454,,,
 AR66,,AR70,,CP04OSSM-SBD12-03-HYDGN0000,15,1,2022-04-09T14:56:00,2022-11-10T13:36:00,CGMCP-04OSSM-00015,,CGINS-HYDGN0-49857,39.937517,-70.88685,,0,454,,,
 AR66,,AR70,,CP04OSSM-SBD12-04-PCO2AA000,15,1,2022-04-09T14:56:00,2022-11-10T13:36:00,CGMCP-04OSSM-00015,,CGINS-PCO2AA-34233,39.937517,-70.88685,,0,454,,,
```
[the remark this answers is not in the document]
Comment (sbeaulieu) [reply deployment/CP04OSSM_Deploy.csv:573]: agree looks good

File: deployment/CP01CNSM_Deploy.csv [hunk 5]
```diff
@@ -61,11 +75,25 @@ KN224,,KN224,,CP01CNSM-SBD11-00-DCLENG000,2,1,2014-12-13T18:47:00,2014-12-15T20:
...
 KN224,,KN224,,CP01CNSM-SBD12-05-WAVSSA000,2,1,2014-12-13T18:47:00,2014-12-15T20:58:00,CGMCP-01CNSM-00002,,CGINS-WAVSSA-05791,40.1339,-70.7789,,0,135,,,
 KN224,,KN224,,CP01CNSM-SBD12-06-METBKA000,2,1,2014-12-13T18:47:00,2014-12-15T20:58:00,CGMCP-01CNSM-00002,,CGINS-METLGR-00015,40.1339,-70.7789,,0,135,,,
+KN224,,KN224,,CP01CNSM-SBD12-06-METBKA001,2,1,2014-12-13T18:47:00,2014-12-15T20:58:00,CGMCP-01CNSM-00002,,CGINS-METCTD-11091,40.1339,-70.7789,,1,135,,,
+KN224,,KN224,,CP01CNSM-SBD12-06-METBKA002,2,1,2014-12-13T18:47:00,2014-12-15T20:58:00,CGMCP-01CNSM-00002,,CGINS-METBPR-00238,40.1339,-70.7789,,-4,135,,,
+KN224,,KN224,,CP01CNSM-SBD12-06-METBKA003,2,1,2014-12-13T18:47:00,2014-12-15T20:58:00,CGMCP-01CNSM-00002,,CGINS-METHRH-00280,40.1339,-70.7789,,-4,135,,,
+KN224,,KN224,,CP01CNSM-SBD12-06-METBKA004,2,1,2014-12-13T18:47:00,2014-12-15T20:58:00,CGMCP-01CNSM-00002,,CGINS-METLWR-00245,40.1339,-70.7789,,-4,135,,,
+KN224,,KN224,,CP01CNSM-SBD12-06-METBKA005,2,1,2014-12-13T18:47:00,2014-12-15T20:58:00,CGMCP-01CNSM-00002,,CGINS-METPRC-00245,40.1339,-70.7789,,-4,135,,,
```
Comment (sbeaulieu) [inline deployment/CP01CNSM_Deploy.csv:94]: I'd like to check two UID's 00245

File: deployment/CP01CNSM_Deploy.csv [hunk 6]
```diff
@@ -95,11 +123,25 @@ AT27,,AT31,,CP01CNSM-SBD11-00-DCLENG000,3,1,2015-05-07T17:34:00,2015-10-23T19:40
...
 AT27,,AT31,,CP01CNSM-SBD12-04-PCO2AA000,3,1,2015-05-07T17:34:00,2015-10-23T19:40:00,CGMCP-01CNSM-00003,,CGINS-PCO2AA-34222,40.14022,-70.77128,,0,135,,,
 AT27,,AT31,,CP01CNSM-SBD12-05-WAVSSA000,3,1,2015-05-07T17:34:00,2015-10-23T19:40:00,CGMCP-01CNSM-00003,,CGINS-WAVSSA-05791,40.14022,-70.77128,,0,135,,,
 AT27,,AT31,,CP01CNSM-SBD12-06-METBKA000,3,1,2015-05-07T17:34:00,2015-10-23T19:40:00,CGMCP-01CNSM-00003,,CGINS-METLGR-00015,40.14022,-70.77128,,0,135,,,
+AT27,,AT31,,CP01CNSM-SBD12-06-METBKA001,3,1,2015-05-07T17:34:00,2015-10-23T19:40:00,CGMCP-01CNSM-00003,,CGINS-METCTD-11091,40.14022,-70.77128,,1,135,,,
+AT27,,AT31,,CP01CNSM-SBD12-06-METBKA002,3,1,2015-05-07T17:34:00,2015-10-23T19:40:00,CGMCP-01CNSM-00003,,CGINS-METBPR-00238,40.14022,-70.77128,,-4,135,,,
+AT27,,AT31,,CP01CNSM-SBD12-06-METBKA003,3,1,2015-05-07T17:34:00,2015-10-23T19:40:00,CGMCP-01CNSM-00003,,CGINS-METHRH-00280,40.14022,-70.77128,,-4,135,,,
+AT27,,AT31,,CP01CNSM-SBD12-06-METBKA004,3,1,2015-05-07T17:34:00,2015-10-23T19:40:00,CGMCP-01CNSM-00003,,CGINS-METLWR-00280,40.14022,-70.77128,,-4,135,,,
```
Comment (sbeaulieu) [inline deployment/CP01CNSM_Deploy.csv:141]: I'd like to check 2 UID's 00280

File: deployment/CP01CNSM_Deploy.csv [hunk 7]
```diff
@@ -129,11 +171,25 @@ AT31,,AR1-07,,CP01CNSM-SBD11-00-DCLENG000,4,1,2015-10-23T18:49:00,2016-04-04T12:
...
+AT31,,AR1-07,,CP01CNSM-SBD12-06-METBKA001,4,1,2015-10-23T18:49:00,2016-04-04T12:03:00,CGMCP-01CNSM-00004,,CGINS-METCTD-13546,40.13323,-70.77843,,1,136,,,
+AT31,,AR1-07,,CP01CNSM-SBD12-06-METBKA002,4,1,2015-10-23T18:49:00,2016-04-04T12:03:00,CGMCP-01CNSM-00004,,CGINS-METBPR-00266,40.13323,-70.77843,,-4,136,,,
+AT31,,AR1-07,,CP01CNSM-SBD12-06-METBKA003,4,1,2015-10-23T18:49:00,2016-04-04T12:03:00,CGMCP-01CNSM-00004,,CGINS-METHRH-00297,40.13323,-70.77843,,-4,136,,,
+AT31,,AR1-07,,CP01CNSM-SBD12-06-METBKA004,4,1,2015-10-23T18:49:00,2016-04-04T12:03:00,CGMCP-01CNSM-00004,,CGINS-METLWR-00292,40.13323,-70.77843,,-4,136,,,
+AT31,,AR1-07,,CP01CNSM-SBD12-06-METBKA005,4,1,2015-10-23T18:49:00,2016-04-04T12:03:00,CGMCP-01CNSM-00004,,CGINS-METPRC-00271,40.13323,-70.77843,,-4,136,,,
+AT31,,AR1-07,,CP01CNSM-SBD12-06-METBKA006,4,1,2015-10-23T18:49:00,2016-04-04T12:03:00,CGMCP-01CNSM-00004,,CGINS-METSWR-00264,40.13323,-70.77843,,-4,136,,,
+AT31,,AR1-07,,CP01CNSM-SBD12-06-METBKA007,4,1,2015-10-23T18:49:00,2016-04-04T12:03:00,CGMCP-01CNSM-00004,,CGINS-METWND-00264,40.13323,-70.77843,,-5,136,,,
```
Comment (sbeaulieu) [inline deployment/CP01CNSM_Deploy.csv:192]: I'd like to check 2 UID's 00264

File: deployment/CP01CNSM_Deploy.csv [hunk 8]
```diff
@@ -163,11 +219,25 @@ AR4,,AR8,,CP01CNSM-SBD11-00-DCLENG000,5,1,2016-05-13T13:50:00,2016-10-13T19:34:0
...
+AR4,,AR8,,CP01CNSM-SBD12-06-METBKA001,5,1,2016-05-13T13:50:00,2016-10-13T19:34:00,CGMCP-01CNSM-00005,,CGINS-METCTD-11116,40.14037,-70.77133,,1,134,,,
+AR4,,AR8,,CP01CNSM-SBD12-06-METBKA002,5,1,2016-05-13T13:50:00,2016-10-13T19:34:00,CGMCP-01CNSM-00005,,CGINS-METBPR-00238,40.14037,-70.77133,,-4,134,,,
+AR4,,AR8,,CP01CNSM-SBD12-06-METBKA003,5,1,2016-05-13T13:50:00,2016-10-13T19:34:00,CGMCP-01CNSM-00005,,CGINS-METHRH-00265,40.14037,-70.77133,,-4,134,,,
+AR4,,AR8,,CP01CNSM-SBD12-06-METBKA004,5,1,2016-05-13T13:50:00,2016-10-13T19:34:00,CGMCP-01CNSM-00005,,CGINS-METLWR-00278,40.14037,-70.77133,,-4,134,,,
+AR4,,AR8,,CP01CNSM-SBD12-06-METBKA005,5,1,2016-05-13T13:50:00,2016-10-13T19:34:00,CGMCP-01CNSM-00005,,CGINS-METPRC-00244,40.14037,-70.77133,,-4,134,,,
+AR4,,AR8,,CP01CNSM-SBD12-06-METBKA006,5,1,2016-05-13T13:50:00,2016-10-13T19:34:00,CGMCP-01CNSM-00005,,CGINS-METSWR-00262,40.14037,-70.77133,,-4,134,,,
+AR4,,AR8,,CP01CNSM-SBD12-06-METBKA007,5,1,2016-05-13T13:50:00,2016-10-13T19:34:00,CGMCP-01CNSM-00005,,CGINS-METWND-00244,40.14037,-70.77133,,-5,134,,,
```
Comment (sbeaulieu) [inline deployment/CP01CNSM_Deploy.csv:240]: separated (non-adjacent) rows but I'd still like to check 2 UID's 00244

File: deployment/CP01CNSM_Deploy.csv [hunk 9]
```diff
@@ -333,11 +459,25 @@ AR31,,AR34,,CP01CNSM-SBD11-00-DCLENG000,10,1,2018-10-30T01:48:00,2019-04-07T18:0
...
 AR31,,AR34,,CP01CNSM-SBD12-06-METBKA000,10,1,2018-10-30T01:48:00,2019-04-07T18:08:00,CGMCP-01CNSM-00010,,CGINS-METLGR-00017,40.1333667,-70.7777,,0,134,,,
+AR31,,AR34,,CP01CNSM-SBD12-06-METBKA001,10,1,2018-10-30T01:48:00,2019-04-07T18:08:00,CGMCP-01CNSM-00010,,CGINS-METCTD-11560,40.1333667,-70.7777,,1,134,,,
+AR31,,AR34,,CP01CNSM-SBD12-06-METBKA002,10,1,2018-10-30T01:48:00,2019-04-07T18:08:00,CGMCP-01CNSM-00010,,CGINS-METBPR-00270,40.1333667,-70.7777,,-4,134,,,
+AR31,,AR34,,CP01CNSM-SBD12-06-METBKA003,10,1,2018-10-30T01:48:00,2019-04-07T18:08:00,CGMCP-01CNSM-00010,,CGINS-METHRH-00261,40.1333667,-70.7777,,-4,134,,,
+AR31,,AR34,,CP01CNSM-SBD12-06-METBKA004,10,1,2018-10-30T01:48:00,2019-04-07T18:08:00,CGMCP-01CNSM-00010,,CGINS-METLWR-00276,40.1333667,-70.7777,,-4,134,,,
+AR31,,AR34,,CP01CNSM-SBD12-06-METBKA005,10,1,2018-10-30T01:48:00,2019-04-07T18:08:00,CGMCP-01CNSM-00010,,CGINS-METPRC-00242,40.1333667,-70.7777,,-4,134,,,
+AR31,,AR34,,CP01CNSM-SBD12-06-METBKA006,10,1,2018-10-30T01:48:00,2019-04-07T18:08:00,CGMCP-01CNSM-00010,,CGINS-METSWR-00242,40.1333667,-70.7777,,-4,134,,,
```
Comment (sbeaulieu) [inline deployment/CP01CNSM_Deploy.csv:479]: I'd like to check 2 UID's 00242

File: deployment/CP01CNSM_Deploy.csv [hunk 10]
```diff
@@ -401,11 +555,25 @@ AR39,,AR48,,CP01CNSM-SBD11-00-DCLENG000,12,1,2019-09-27T18:30:00,2020-11-06T13:1
 AR39,,AR48,,CP01CNSM-SBD11-01-MOPAK0000,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGINS-MOPAK0-11843,40.1332,-70.7783,,0,133,,,
 AR39,,AR48,,CP01CNSM-SBD11-02-HYDGN0000,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGINS-HYDGN0-42330,40.1332,-70.7783,,0,133,,,
 AR39,,AR48,,CP01CNSM-SBD11-06-METBKA000,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGINS-METLGR-00010,40.1332,-70.7783,,0,133,,,
+AR39,,AR48,,CP01CNSM-SBD11-06-METBKA001,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGINS-METCTD-11091,40.1332,-70.7783,,1,133,,,
+AR39,,AR48,,CP01CNSM-SBD11-06-METBKA002,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGINS-METBPR-00272,40.1332,-70.7783,,-4,133,,,
+AR39,,AR48,,CP01CNSM-SBD11-06-METBKA003,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGINS-METHRH-00261,40.1332,-70.7783,,-4,133,,,
+AR39,,AR48,,CP01CNSM-SBD11-06-METBKA004,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGINS-METLWR-00272,40.1332,-70.7783,,-4,133,,,
```
Comment (sbeaulieu) [inline deployment/CP01CNSM_Deploy.csv:561]: non-adjacent 2 UID's 00272

File: deployment/CP01CNSM_Deploy.csv [hunk 11]
```diff
@@ -435,11 +603,25 @@ AR48,,AR52,,CP01CNSM-SBD11-00-DCLENG000,13,1,2020-10-29T14:57:00,2021-04-03T12:0
...
 AR48,,AR52,,CP01CNSM-SBD11-06-METBKA000,13,1,2020-10-29T14:57:00,2021-04-03T12:05:00,CGMCP-01CNSM-00013,,CGINS-METLGR-00003,40.140517,-70.769643,,0,133,,,
+AR48,,AR52,,CP01CNSM-SBD11-06-METBKA001,13,1,2020-10-29T14:57:00,2021-04-03T12:05:00,CGMCP-01CNSM-00013,,CGINS-METCTD-11116,40.140517,-70.769643,,1,133,,,
+AR48,,AR52,,CP01CNSM-SBD11-06-METBKA002,13,1,2020-10-29T14:57:00,2021-04-03T12:05:00,CGMCP-01CNSM-00013,,CGINS-METBPR-00256,40.140517,-70.769643,,-4,133,,,
+AR48,,AR52,,CP01CNSM-SBD11-06-METBKA003,13,1,2020-10-29T14:57:00,2021-04-03T12:05:00,CGMCP-01CNSM-00013,,CGINS-METHRH-00289,40.140517,-70.769643,,-4,133,,,
+AR48,,AR52,,CP01CNSM-SBD11-06-METBKA004,13,1,2020-10-29T14:57:00,2021-04-03T12:05:00,CGMCP-01CNSM-00013,,CGINS-METLWR-00295,40.140517,-70.769643,,-4,133,,,
+AR48,,AR52,,CP01CNSM-SBD11-06-METBKA005,13,1,2020-10-29T14:57:00,2021-04-03T12:05:00,CGMCP-01CNSM-00013,,CGINS-METPRC-00246,40.140517,-70.769643,,-4,133,,,
+AR48,,AR52,,CP01CNSM-SBD11-06-METBKA006,13,1,2020-10-29T14:57:00,2021-04-03T12:05:00,CGMCP-01CNSM-00013,,CGINS-METSWR-00246,40.140517,-70.769643,,-4,133,,,
```
Comment (sbeaulieu) [inline deployment/CP01CNSM_Deploy.csv:611]: I'd like to check 2 UID's 00246

File: deployment/CP03ISSM_Deploy.csv (hunk 2 above)
Comment (sbeaulieu) [reply deployment/CP03ISSM_Deploy.csv:188]: looks good

File: deployment/CP03ISSM_Deploy.csv [hunk 12]
```diff
@@ -369,6 +446,13 @@ AR48,,AR52,,CP03ISSM-SBD11-00-DCLENG000,12,1,2020-10-31T19:31:00,2021-03-31T19:0
...
+AR48,,AR52,,CP03ISSM-SBD11-06-METBKA001,12,1,2020-10-31T19:31:00,2021-03-31T19:02:00,CGMCP-03ISSM-00012,,CGINS-METCTD-11561,40.367077,-70.881735,,1,91,,,
+AR48,,AR52,,CP03ISSM-SBD11-06-METBKA002,12,1,2020-10-31T19:31:00,2021-03-31T19:02:00,CGMCP-03ISSM-00012,,CGINS-METBPR-00267,40.367077,-70.881735,,-4,91,,,
+AR48,,AR52,,CP03ISSM-SBD11-06-METBKA003,12,1,2020-10-31T19:31:00,2021-03-31T19:02:00,CGMCP-03ISSM-00012,,CGINS-METHRH-00298,40.367077,-70.881735,,-4,91,,,
+AR48,,AR52,,CP03ISSM-SBD11-06-METBKA004,12,1,2020-10-31T19:31:00,2021-03-31T19:02:00,CGMCP-03ISSM-00012,,CGINS-METLWR-00292,40.367077,-70.881735,,-4,91,,,
+AR48,,AR52,,CP03ISSM-SBD11-06-METBKA005,12,1,2020-10-31T19:31:00,2021-03-31T19:02:00,CGMCP-03ISSM-00012,,CGINS-METPRC-00236,40.367077,-70.881735,,-4,91,,,
+AR48,,AR52,,CP03ISSM-SBD11-06-METBKA006,12,1,2020-10-31T19:31:00,2021-03-31T19:02:00,CGMCP-03ISSM-00012,,CGINS-METSWR-00264,40.367077,-70.881735,,-4,91,,,
+AR48,,AR52,,CP03ISSM-SBD11-06-METBKA007,12,1,2020-10-31T19:31:00,2021-03-31T19:02:00,CGMCP-03ISSM-00012,,CGINS-METWND-00264,40.367077,-70.881735,,-5,91,,,
```
[the remark this answers is not in the document]
Comment (sbeaulieu) [reply deployment/CP03ISSM_Deploy.csv:455]: looks good

File: deployment/CP03ISSM_Deploy.csv (hunk 3 above)
Comment (sbeaulieu) [reply deployment/CP03ISSM_Deploy.csv:529]: looks good

File: deployment/CP01CNSM_Deploy.csv (hunk 5 above)
Comment (sbeaulieu) [reply deployment/CP01CNSM_Deploy.csv:94]: looks good

File: deployment/CP01CNSM_Deploy.csv (hunk 7 above)
Comment (sbeaulieu) [reply deployment/CP01CNSM_Deploy.csv:192]: looks good

File: deployment/CP01CNSM_Deploy.csv (hunk 8 above)
Comment (sbeaulieu) [reply deployment/CP01CNSM_Deploy.csv:240]: looks good

File: deployment/CP01CNSM_Deploy.csv (hunk 9 above)
Comment (sbeaulieu) [reply deployment/CP01CNSM_Deploy.csv:479]: looks good

File: deployment/CP01CNSM_Deploy.csv (hunk 10 above)
Comment (sbeaulieu) [reply deployment/CP01CNSM_Deploy.csv:561]: looks good

File: deployment/CP01CNSM_Deploy.csv (hunk 11 above)
Comment (sbeaulieu) [reply deployment/CP01CNSM_Deploy.csv:611]: looks good

**Review by sbeaulieu — APPROVED** [review]

**Pull request merged**